repository/Seaside-Canvas.package/WAScriptTag.class/instance/with..st

```diff
@@ -1,9 +1,13 @@
 public
-with: aString
+with: aStringOrJSObject
 
 	self attributes
 		at: 'type'
 		ifAbsentPut: [ 'text/javascript' ].
 	super with: [
-		aString ifNotNil: [
-			self document nextPutAll: aString greaseString ] ]
\ No newline at end of file
+		aStringOrJSObject ifNotNil: [
+			aStringOrJSObject isString
```

Comment:
Maybe `isJavascript` is better since it's a method defined in the Seaside codebase, while we otherwise rely on `isString` to be defined on all classes that implement strings. Perhaps that is also trivially true though (or should be).